convex/README.md

```diff
@@ -82,6 +85,6 @@ function handleButtonPress() {
 }
 ```
 
-The Convex CLI is your friend. See everything it can do by running
-`npx convex -h` in your project root directory. To learn more, launch the docs
-with `npx convex docs`.
```

Comment:
Maybe we should remove this file, as it might distract from the rest of the code.

Reply:
Or maybe it should live in _generated?
All templates have it right now, but I'm open to hearing a case for removing. It is a handy thing to copy/paste since we have so much boilerplate now. I personally use snippets but anecdotally other employees have just used it to copy-paste